addons/strings/stringtable.xml

```diff
@@ -32,12 +32,7 @@
         </Key>
         <Key ID="STR_CBA_FORMAT_NUMBER_THOUSANDS_SEPARATOR">
             <English>,</English>
-            <Czech> </Czech>
             <German>.</German>
-            <Polish> </Polish>
-            <Russian> </Russian>
-            <French> </French>
-            <Spanish> </Spanish>
```

Comment:
Restore this file please.

Reply:
I'm sorry, it seems tabler deleted itself the "empty" lines !
Done ;) !

Reply:
Np, but I hthink if you update Tabler it should no longer do this.
@bux?